src/lib/utils/decode/module-decoding.go

```diff
@@ -0,0 +1,68 @@
+package decode
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/zairza-cetb/bench-routes/src/lib/modules/jitter"
+	"github.com/zairza-cetb/bench-routes/src/lib/modules/ping"
+	"github.com/zairza-cetb/bench-routes/src/metrics/system"
```

Comment:
implement this package as a form of an interface as it was earlier in the decoder.